pkg/rancher-desktop/pages/containers/ContainerInfo.vue

```diff
@@ -0,0 +1,386 @@
+<template>
+  <div
+    class="container-info-page"
+    data-testid="container-info"
+  >
+    <div class="tab-header-row">
+      <ul class="tabs">
+        <li
+          class="tab active"
+          data-testid="tab-logs"
+        >
+          <a>
+            <span>Logs</span>
+          </a>
+        </li>
+      </ul>
+      <div class="search-widget">
+        <input
+          ref="searchInput"
+          v-model="searchTerm"
+          aria-label="Search in logs"
+          class="search-input"
+          data-testid="search-input"
+          placeholder="Search logs..."
+          type="search"
+          @input="onSearchInput"
+          @keydown="handleSearchKeydown"
+        >
+        <button
+          :disabled="!searchTerm"
+          aria-label="Previous match"
+          class="search-btn btn role-tertiary"
+          data-testid="search-prev-btn"
+          title="Previous match"
+          @click="searchPrevious"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-chevron-up"
+          />
+        </button>
+        <button
+          :disabled="!searchTerm"
+          aria-label="Next match"
+          class="search-btn btn role-tertiary"
+          data-testid="search-next-btn"
+          title="Next match"
+          @click="searchNext"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-chevron-down"
+          />
+        </button>
+        <button
+          :disabled="!searchTerm"
+          aria-label="Clear search"
+          class="search-btn btn role-tertiary"
+          data-testid="search-clear-btn"
+          title="Clear search"
+          @click="clearSearch"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-x"
+          />
+        </button>
+      </div>
+    </div>
+    <div class="tab-content">
+      <container-logs
+        v-if="containerId"
+        ref="containerLogs"
+        :container-id="containerId"
+        :is-container-running="isRunning"
+        :namespace="settings?.containers?.namespace"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue';
+import { mapGetters } from 'vuex';
```

Comment:
Since we already use `mapTypedState`, might as well use `mapTypedGetters` from the same file.  The only difference is that it lets TypeScript figure out the types of the mapped values.

(Using `<script lang="ts">` is preferred in new files; it would be good if this could use that. But don't bother if it's more work than expected.)